packages/ckb-sdk-core/src/index.ts

```diff
@@ -161,6 +200,55 @@ class Core {
       witnesses: signedWitnesses,
     }
   }
+
+  public generateRawTransaction = async ({
```

Comment:
Generally speaking building a raw transaction should not have any dependency on the knowledge of mainnet/testnet. I understand that the `isMainnet` is used to pass to `utils` to parse address to public key hash, but this pollutes the interface of `generateRawTransaction` which is no good.

The small issue here is its implementation depends on `utils.parseAddress` which needs to know if it's mainnet/testnet. 

One way to get around this is to change `utils.parseAddress` to not require passing in `prefix`. We're mostly certain CKB would only have `ckb` and `ckt` prefixes, and parsing an address doesn't need to ask for the prefix to be passed in - the parsing could just take the first three characters.

Reply:
> Generally speaking building a raw transaction should not have any dependency on the knowledge of mainnet/testnet. I understand that the `isMainnet` is used to pass to `utils` to parse address to public key hash, but this pollutes the interface of `generateRawTransaction` which is no good.
> 
> The small issue here is its implementation depends on `utils.parseAddress` which needs to know if it's mainnet/testnet.
> 
> One way to get around this is to change `utils.parseAddress` to not require passing in `prefix`. We're mostly certain CKB would only have `ckb` and `ckt` prefixes, and parsing an address doesn't need to ask for the prefix to be passed in - the parsing could just take the first three characters.

Yes, parsing an address doesn't require the `prefix`, and in the implementation of `parseAddress` it's just used to guarantee the address is working on the explicitly specified blockchain. 

I will remove it from the parameter list.

Reply:
`prefix` is removed from the parameter list in https://github.com/nervosnetwork/ckb-sdk-js/pull/356/commits/9e4477ca39692512fa3be4099f387c8ef396a79b